aws/resource_aws_codedeploy_deployment_group.go

```diff
@@ -670,6 +1015,21 @@ func resourceAwsCodeDeployTriggerConfigHash(v interface{}) int {
 	return hashcode.String(buf.String())
 }
 
+func loadBalancerInfoHash(v interface{}) int {
```

Comment:
This custom hash function shouldn't be necessary as the nested schema seems quite simple, so the default one (which is used when `Set` is not defined) should suffice.